naughty/rhel-9/2840-kdump-no-memory

```diff
@@ -0,0 +1,4 @@
+testBasic (__main__.TestKdump)
+Job for kdump.service failed because the control process exited with error code.
```

Comment:
This feels brittle, like this is missing a `*` after the first line (so that there can be zero to arbitrarily many lines in between). But if it works, sure!

Reply:
Seems to work :+1: